src/stories/file/FileWithDeniedPermissionsRepository.ts

```diff
@@ -1,17 +1,17 @@
 import { FileRepository } from '../../files/domain/repositories/FileRepository'
-import { FileUserPermissionsMother } from '../../../tests/component/files/domain/models/FileUserPermissionsMother'
-import { FileUserPermissions } from '../../files/domain/models/FileUserPermissions'
+import { FilePermissionsMother } from '../../../tests/component/files/domain/models/FilePermissionsMother'
+import { FilePermissions } from '../../files/domain/models/FilePermissions'
 import { FileMockRepository } from './FileMockRepository'
 
 export class FileWithDeniedPermissionsRepository
   extends FileMockRepository
   implements FileRepository
 {
   // eslint-disable-next-line unused-imports/no-unused-vars
-  getUserPermissionsById(id: number): Promise<FileUserPermissions> {
+  getUserPermissionsById(id: number): Promise<FilePermissions> {
```

Comment:
We can delete this method since we removed getUserPermissionsById from the FileRepository